streamalert_cli/terraform/firehose.py

```diff
@@ -46,32 +50,35 @@ def generate_firehose(logging_bucket, main_dict, config):
     }
 
     enabled_logs = FirehoseClient.load_enabled_log_sources(
-        config['global']['infrastructure']['firehose'],
+        firehose_conf,
         config['logs'],
         force_load=True
     )
 
-    log_alarms_config = config['global']['infrastructure']['firehose'].get('enabled_logs', {})
+    log_alarms_config = firehose_conf.get('enabled_logs', {})
+
+    db_name = get_database_name(config)
 
     # Add the Delivery Streams individually
     for log_stream_name, log_type_name in enabled_logs.items():
         module_dict = {
             'source': './modules/tf_kinesis_firehose_delivery_stream',
             'buffer_size': (
-                config['global']['infrastructure']['firehose'].get('buffer_size', 64)
+                firehose_conf.get('buffer_size', 128)
```

Comment:
Any particular reason to change the default?

Reply:
Also, (not related to your code) I would like to eventually move away from these inline defaults. I would like more instances eventually where a missing configuration simply raises the `MisconfigurationError` that you added

Reply:
Checked, we have default value `5` in `terraform/modules/tf_kinesis_firehose_delivery_stream/variables.tf` and `128` value in `conf/global.json`. So take Ryan's suggestion, I remove the default value here.